package.json

```diff
@@ -147,7 +147,12 @@
         "category": "ONE"
       },
       {
-        "command": "one.explorer.reveal",
+        "command": "one.explorer.revealInOneExplorer",
+        "title": "Reveal in ONE Explorer View",
+        "category": "ONE"
+      },
+      {
+        "command": "one.explorer.revealInDefaultExplorer",
```

Comment:
NOTE: I changed the name 'one.explorer.reveal' command into 'revealInDefaultExplorer' to distinguish it from 'revealInOneExplorer' command.